### PR TITLE
add info about install script

### PR DESCRIPTION
**Pull request by enoch85**
Signed-off-by: Daniel Hansson <daniel@techandme.se>

Feeedback is welcome!

### CODE REVIEW

**Comment by enoch85** [comment]
cc @mario

**Comment by enoch85** [comment]
Can someone re-run the check please?

File: README.md [hunk 1]
```diff
@@ -17,6 +17,16 @@ And in the works for the [coming versions](https://github.com/nextcloud/spreed/m
 
 If you have suggestions or problems, please [open an issue](https://github.com/nextcloud/spreed/issues) or contribute directly :)
 
+
+### Installing
+
+There are several ways of installing Talk. If you just need to enable local access it's enough to just enable the app from the Nextcloud App Store.
```
Comment (Ivansss) [inline README.md]: Nextcloud Talk is really easy to install. You just need to enable the app from the [Nextcloud App Store](https://apps.nextcloud.com/apps/spreed) and everything will work out of the box.

File: README.md [hunk 2]
```diff
@@ -17,6 +17,16 @@ And in the works for the [coming versions](https://github.com/nextcloud/spreed/m
 
 If you have suggestions or problems, please [open an issue](https://github.com/nextcloud/spreed/issues) or contribute directly :)
 
+
+### Installing
+
+There are several ways of installing Talk. If you just need to enable local access it's enough to just enable the app from the Nextcloud App Store. 
+
+If you need to use Talk from outside your own LAN, or through a strict firewall you need to install and setup a TURN server. That's a bit more tricky, but the guys from [Nextcloud VM](https://github.com/nextcloud/vm) has developed a script which takes care of everything for you. You can find the script [here](https://github.com/nextcloud/vm/blob/master/apps/talk.sh). The script is tested on Ubuntu Server 18.04, but should work on 16.04 as well. Please keep in mind that it's developed for the VM specifically and any issues should be reported in that repo, not here.
```
Comment (Ivansss) [inline README.md]: There are some scenarios (users behind strict firewalls / symmetric NATs) where a TURN server is needed. That's a bit more tricky installation, but the guys...

File: README.md [hunk 3]
```diff
@@ -17,6 +17,16 @@ And in the works for the [coming versions](https://github.com/nextcloud/spreed/m
...
+
+**Here's a short video on how it's done:**<br>
+[![Install Talk on Nextcloud](https://lh3.googleusercontent.com/crCv9cBtaOz-5BqpXp0Dhxjq3kyh5rbg0oKx2_BlCwZe2i3nuGhkK2zIzzdCMXFVal8=s180)](https://youtu.be/KdTsWIy4eN0)
```
Comment (Ivansss) [inline README.md]: Here's a short [video](https://youtu.be/KdTsWIy4eN0) on how it's done.

**Comment by Ivansss** [comment]
Thank you so much for your contribution and for the script @enoch85 :)
I would just change the wording a bit with the comments above.

File: README.md (hunk 1 above)
Comment (enoch85) [reply README.md]: @Ivansss Everything won't work out of the box as you need Turn for it to work with video calls. But yeah sure, I'll change it.

**Comment by Ivansss** [comment]
Thanks again! 😊

**Pull request merged**